lib/modules/platform/github/index.ts

```diff
@@ -866,6 +879,13 @@ export async function getBranchStatus(
         conclusion: run.conclusion,
       }));
       logger.debug({ checkRuns }, 'check runs result');
+      if (!internalChecksAsSuccess) {
+        checkRuns = checkRuns.filter(
```

Comment:
don't we only post renovate as status checks? so this filter never removes any check run?

Reply:
true